src/map/items/item_armor.cpp

```diff
@@ -255,6 +255,21 @@ void CItemArmor::LoadAugment(uint8 slot, uint16 augment)
     ref<uint16>(m_extra, 2 + (slot * 2)) = augment;
 }
 
+void CItemArmor::PushAugment(uint16 type, uint8 value)
+{
+    uint8 slot = 0;
+    uint16 augment = ref<uint16>(m_extra, 2 + (slot * 2));
+    while (augment != 0 && slot < 4)
```

Comment:
i guess we should enum max augment slot somewhere in case it changed in the future